src/main/resources/application.conf

```diff
@@ -32,27 +18,36 @@ spray.can {
   }
 }
 
+workflow-options {
+  // These workflow options will be encrypted when stored in the database
+  encrypted-fields: ["refresh_token"]
+
+  // AES-256 key to use to encrypt the values in `encrypted-fields`
+  base64-encryption-key: "q78yJ9CtbWTX3+zDHJgHSiN4I7+nq5kgYjOD94nWK48="
```

Comment:
This was just there for some manual testing, I forgot to remove it.  Though I should test that it also behaves the right way if there is no encryption key when it's expecting one.